packages/shared/scope-hoisting/src/prelude.js

```diff
@@ -1,8 +1,19 @@
 var $parcel$modules = {};
 var $parcel$bundles = {};
 
-if (parcelRequire == null) {
-  parcelRequire = function(name) {
+var $parcel$global =
+  typeof globalThis !== 'undefined'
+    ? globalThis
+    : typeof self !== 'undefined'
+    ? self
+    : typeof window !== 'undefined'
+    ? window
+    : typeof global !== 'undefined'
+    ? global
+    : {};
```

Comment:
I couldn't find a way to reuse the one in helpers.js because that one will be inside the `$parcel$bundleWrapper`